keps/sig-node/3085-pod-conditions-for-starting-completition-of-sandbox-creation/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: KEP Template
+kep-number: 3085
+authors:
+  - "@ddebroy"
+  - "@agamdua"
+owning-sig: sig-node
+participating-sigs:
+status: provisional
```

Comment:
Needs to be set to "implementable" in order to target this release

Reply:
thanks - set to "implementable"